persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -162,7 +164,11 @@ Conversation.team_id,
 Conversation.mls_proposal_timer,
 Conversation.muted_time,
 Conversation.creator_id,
-Conversation.last_modified_date
+Conversation.last_modified_date,
```

Comment:
Do we need this field anymore? since we're fetching the message in another transaction?

Reply:
I will check this in next PR